core/src/main/java/org/apache/accumulo/core/client/impl/AccumuloClientImpl.java

```diff
@@ -125,6 +125,13 @@ public BatchScanner createBatchScanner(String tableName, Authorizations authoriz
     return createBatchScanner(tableName, authorizations, numQueryThreads);
   }
 
+  @Override
+  public BatchScanner createBatchScanner(String tableName) throws TableNotFoundException,
+      AccumuloSecurityException, AccumuloException {
+    Authorizations auths = securityOperations().getUserAuthorizations(context.getPrincipal());
```

Comment:
See comment above. This is okay for a first pass implementation, but not sufficient with pluggable authorization providers which could have an infinite set of auths.